.gitignore

```diff
@@ -4,13 +4,18 @@
 
 # Build
 # TODO: Adjust binary file name
-/go-bootstrap
+/exoscale-metrics-collector
 *.out
+/component/compiled
+/component/vendor
 
 # Docs
 /.cache/
 /.public/
 /node_modules/
 
-# work
-/.work/
+# IDE
+.idea/
```

Comment:
FYI: You might want add this one to your global ~/.gitignore so we don't have to list every IDE in every project.

Reply:
Done.